README.md

```diff
@@ -1,25 +1,21 @@
-<p align="center">
-<a href="https://prow.build-infra.jetstack.net/?job=post-preflight-release-canary">
-<!-- prow build badge, godoc, and go report card-->
-<img alt="Build Status" src="https://prow.build-infra.jetstack.net/badge.svg?jobs=post-preflight-release-canary">
-</a>
-<a href="https://godoc.org/github.com/jetstack/preflight"><img src="https://godoc.org/github.com/jetstack/preflight?status.svg"></a>
-<a href="https://goreportcard.com/report/github.com/jetstack/preflight"><img alt="Go Report Card" src="https://goreportcard.com/badge/github.com/jetstack/preflight" /></a>
-</p>
-
-# Jetstack Preflight
-
-Preflight is a tool to automatically perform Kubernetes cluster configuration
+# Jetstack Secure Agent
```

Comment:
The name of the repo is still preflight. So I would add a note explaining that preflight is now part of Jetstack Secure.